docs/openapi.yml

```diff
@@ -1028,6 +1028,10 @@ components:
     newRun:
       type: object
       properties:
+        runId:
+          description: An optional client-provided UUID for this run (this could be a Random or name based UUID). If not provided a random UUID will be generated for the run
+          type: string
+          format: uuid
```

Comment:
You'll also want to render the `html` using `redoc-cli bundle`, see https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#api-docs

Reply:
thanks for reminding me.